index.js

```diff
@@ -72,8 +73,12 @@ const receivePhoneCall = async (event, { logger, csClient,storageClient } ) => {
   // const connected_agents = await storageClient.get('connected_users') || ""
```

Comment:
please remove the CONNECTED_USERS, var, you shoud uncomment storageClient.get('connected_users') || "" and use that

Reply:
Done